sqlpuzzle/_common/sqlvalue.py

```diff
@@ -86,6 +87,9 @@ def _string(self):
             return self._reference()
         return "'{}'".format(_escape_value(force_text(self.value)))
 
+    def _bytes(self):
+        return "x'{}'".format(self.value.hex())
```

Comment:
Why hex? Is it working for all databases or is it database specific?

Reply:
Oops, looks like it's specific for Mysql. I will try to find general solution.

Reply:
Now it's working. Tested live against all backends.